server/server.ts

```diff
@@ -131,12 +131,28 @@ app.post('/DEVchangeDir', (req, res) => {
 
 app.listen(PORT, () => {
   console.clear();
-  console.log(`Listening on port ${PORT}!`);
+  startingASCII();
+  // TODO: Add version/build number here
+  console.log('Listening on ', colors.grey(`http://localhost:${PORT}`));
   if (isOpenBrowser) {
     openBrowser(PORT);
   }
 });
 
+function startingASCII() {
```

Comment:
move this function to a helper.
could be called something like `ascii-logo-printer.helper.ts`
then export and import it.